src/components/ButtonPanel/ButtonPanel.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react';
+import './ButtonPanel.css'
+import Button from "../Button/Button";
+import PropTypes from "prop-types";
+
+class ButtonPanel extends Component {
```

Comment:
Let's make sure all components that are not keeping/using state and not using any lifecycle method are just written as stateless functional components. Using `class` for such components adds unnecessary baggages in the component (makes it slower to mount as it needs to call all those pre-mounting lifecycle methods).